packages/path_provider/path_provider_linux/pubspec.yaml

```diff
@@ -16,12 +16,12 @@ flutter:
         dartPluginClass: PathProviderLinux
 
 dependencies:
-  ffi: ">=1.1.2 <3.0.0"
+  ffi: ">=2.0.1"
```

Comment:
This removes compatibility with `ffi` 1.x without any benefit, creating more potential conflicts in the ecosystem.